drivers/watchdog/Kconfig.sdw_apb

```diff
@@ -0,0 +1,21 @@
+# Synopsys DW AHB WDOG cfg
+
+# Copyright 2022 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+
+config WDOG_SDW_APB
+	bool "SDW APB Watchdog Driver for RISCV family of MCUs"
```

Comment:
WDOG -> WDT is more suitable, I think.
Newer drivers are commonly using WDT prefix.